packages/oslo-generator-shacl-template/lib/handlers/PropertyShapeBaseHandler.ts

```diff
@@ -218,4 +263,48 @@ export class PropertyShapeBaseHandler extends ShaclHandler {
 
     super.handle(subject, store, shaclStore);
   }
+
+  private discoverRangeClasses(
```

Comment:
This must be ported as well to CLassShapeBaseHandler.ts to avoid missing EA objects with different GUIDs but the same assigned URI.